public/app/plugins/datasource/testdata/components/RandomWalkEditor.tsx

```diff
@@ -27,7 +27,7 @@ export const RandomWalkEditor = ({ onChange, query }: EditorProps) => {
               width={32}
               name={id}
               type="number"
-              id={id}
+              id={`${id}-${query.refId}`}
```

Comment:
```suggestion
              id={`randomWalk-${id}-${query.refId}`}
```
the others all prefix with scenarioId